### ISSUE TITLE
OSX 8 still has JDK 6 available

### ISSUE THREAD

**Issue by ascheman** [takari/maven-wrapper#18]
On OSX 8 (10.8.x) there is still a JDK 6 provided in `/System/Library/Frameworks/JavaVM.framework/Versions/CurrentJDK`. Since `.mvn/wrapper/maven-wrapper.jar` was compiled with JDK 7, it cannot be loaded but raises the following exception: 

```
Exception in thread "main" java.lang.UnsupportedClassVersionError: org/apache/maven/wrapper/MavenWrapperMain : Unsupported major.minor version 51.0
```

I see two solutions to the problem:
1. Change the search order in the mvnw wrapper script: search for newer Oracle JDKs (7 or 8) first since they can handle this (then completely drop JDK 6 for Darwin since it makes no sense!)
2. Compile the Java classes with JDK 6 to enable running with older JDKs

I would prefer solution No. 2 since this would also allow to use the wrapper for legacy projects which rely on JDK 6 (yes, there are still some users of this out there in the dark!)

**Comment by abelsromero** [comment]
I am finding the same issue. I compiled the project setting the maven compiler to be compatible with 1.6 and it worked.

```
      <plugin>
          <groupId>org.apache.maven.plugins</groupId>
          <artifactId>maven-compiler-plugin</artifactId>
          <configuration>
              <source>1.6</source>
              <target>1.6</target>
          </configuration>
      </plugin>
```

I know 1.6 is outdated, but I thing it could be done. The changes are minnor and it will help those arround still stuck with 1.6. wdyt?

**Comment by mosabua** [comment]
I believe this PR  https://github.com/takari/maven-wrapper/pull/14 fixes the issue.

**Comment by abelsromero** [comment]
In the end I have created a fork and made the changes in my repo. TBH seems this is unmaintained.

**Comment by mosabua** [comment]
Fixed with upcoming 0.2.0
